src/backends/pyplot.jl

```diff
@@ -824,16 +839,7 @@ function py_set_lims(ax, sp::Subplot, axis::Axis)
     getproperty(ax, Symbol("set_", letter, "lim"))(lfrom, lto)
 end
 
-function py_surround_latextext(latexstring, env)
-    if !isempty(latexstring) && latexstring[1] == '$' && latexstring[end] == '$'
-        unenclosed = latexstring[2:(end - 1)]
-    else
-        unenclosed = latexstring
-    end
-    PyPlot.LaTeXStrings.latexstring(env, "{", unenclosed, "}")
```

Comment:
Will this PR remove support for `LaTeXStrings` ?

Reply:
Nope, all the texts could be latex, matplotlib has its own mathjax like text renderer.